backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -151,12 +151,7 @@ private async Task<bool> WriteDelegationPolicyInternal(string policyPath, List<P
                     return false;
```

Comment:
Logger warning must be updated and appPolicy variable should be renamed resourcePolicy